tools/ci/tc/decision.py

```diff
@@ -100,6 +102,19 @@ def get_extra_jobs(event):
     return jobs
 
 
+def filter_jobs(jobs: Set, event: Dict) -> None:
```

Comment:
Flake8 is upset about this line not using typing.Set and typing.Dict, but these are [deprecated as of Python 3.9](https://docs.python.org/3/library/typing.html#typing.Set). I'm just going to remove the type hints.